source/declarations.tex

```diff
@@ -1779,7 +1779,8 @@
 \opt{\grammarterm{type-constraint}} \keyword{auto}.
 The placeholder type shall appear
 as one of the \grammarterm{decl-specifier}{s} in
-the \grammarterm{decl-specifier-seq} or
+the \grammarterm{decl-specifier-seq} or,
+in case of function pointer type,
```

Comment:
I'm not convinced this is a good change. At the very least, reference-to-function types should be included, too.

Reply:
Proposed a new way, not sure whether for p5 it was really needed to repeat the same note.